app/src/components/App.tsx

```diff
@@ -8,52 +7,54 @@ import { ArtifactView, ProposalView, ARRView, RegisterView, RegisterArtistView,
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 import LeftSidebar from './LeftSidebar';
 
-const App: React.FC = () => {
+import Web3 from 'web3';
+
+interface AppProps {
```

Comment:
You should extend from CommonProps instead of having contracts and accounts duplicated in new props interfaces like this